planet/cli/data.py

```diff
@@ -78,6 +81,18 @@ def check_item_types(ctx, param, item_types) -> Optional[List[dict]]:
         raise click.BadParameter(str(e))
 
 
+def check_item_type(ctx, param, item_type) -> Optional[List[dict]]:
+    '''Validates the item type provided by comparing it to all supported
+    item types.'''
+    if len(item_type.split(",")) > 1:
+        raise click.BadParameter("Only provide 1 item type.")
```

Comment:
OK so I tried this:

```
> planet data asset-activate PSScene,SkySatScene 20221003_002705_38_2461 basic_udm2
Usage: planet data asset-activate [OPTIONS] ITEM_TYPE ITEM_ID ASSET_TYPE_ID
Try 'planet data asset-activate --help' for help.

Error: Invalid value for 'ITEM_TYPE': Only provide 1 item type.
```
And it's not totally clear to me that it is the comma in my value for item-type that caused the issue here. 

When I remove these lines of code and run it again I get:

```
> planet data asset-activate PSScene,SkySatScene 20221003_002705_38_2461 basic_udm2
Usage: planet data asset-activate [OPTIONS] ITEM_TYPE ITEM_ID ASSET_TYPE_ID
Try 'planet data asset-activate --help' for help.

Error: Invalid value for 'ITEM_TYPE': item_type - 'PSScene,SkySatScene' is not one of 'MYD09GQ', 'MYD09GA', 'MOD09GA', 'Landsat8L1G', 'REOrthoTile', 'Sentinel2L1C', 'SkySatCollect', 'SkySatScene', 'REScene', 'PSScene', 'MOD09GQ', 'PSOrthoTile', 'Sentinel1', 'PSScene4Band', 'PSScene3Band'.
```
and to be honest, the second error code gives me more of the information I need to fix the problem.

So my recc here would be to just delete this check.

```suggestion
```